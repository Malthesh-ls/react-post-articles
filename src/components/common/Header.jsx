function Header() {
  return (
    <div className="header">
      <h3>Welcome to React Post Articles</h3>
    </div>
  );
}

export default Header;
