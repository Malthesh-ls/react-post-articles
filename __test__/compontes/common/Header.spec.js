import renderer from 'react-test-renderer';
import Header from '../../../src/components/common/Header';

describe('Header Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<Header />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
