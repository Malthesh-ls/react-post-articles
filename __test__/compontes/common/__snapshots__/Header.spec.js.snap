// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests should match snapshot 1`] = `
<div
  className="header"
>
  <h3>
    Welcome to React Post Articles
  </h3>
</div>
`;
